sql/item_usable.sql

```diff
@@ -1694,6 +1694,7 @@ INSERT INTO `item_usable` VALUES (5892,'b.e.w._pitaru',1,1,28,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (5907,'winterflower',1,1,28,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (5908,'butterpear',1,1,28,0,0,0,0,0);
 INSERT INTO `item_usable` VALUES (5910,'heavy_metal_pouch',1,1,55,0,0,0,0,0);
+INSERT INTO `item_usable` VALUES (5911,'olde_rarab_tail',4,1,55,0,0,0,0,0); -- Need to verify animation
```

Comment:
Would probably want to hold off until this is confirmed, but if there aren't any boned being warped then this is _probably fine_.

Reply:
animation is just the shiny particles that are being assigned. I think it should be 0 as nothing shows on atori when being used. See https://youtu.be/usNZOGDq5c0?t=2682 but I can't confirm as I been looking at the packet that IBM captured but dunno which packet to look for this animation :cry: